app/gui/src/controller/searcher.rs

```diff
@@ -154,6 +154,24 @@ impl Default for Actions {
 
 
 
+// ======================
+// === RequiredImport ===
+// ======================
+
+/// An import that is needed for the picked suggestion.
+#[derive(Debug, Clone, Default)]
+pub enum RequiredImport {
+    /// No import needed.
+    #[default]
+    None,
```

Comment:
Why are we not reusing `Option` here? Option has so many utils defined that in (almost) all such cases its just a better option :) Also, it would make some code easier to write, like the one `.flat_map(|requirement| match requirement [...]` below.

Reply:
I don't have a strong preference here. The code would be a bit longer in signatures, but we can save some space elsewhere. I'll change it.